tests/src/main/java/org/hibernate/beanvalidation/tck/tests/methodvalidation/constraint/ValidStockItemRecord.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Jakarta Validation TCK
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.beanvalidation.tck.tests.methodvalidation.constraint;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import jakarta.validation.Payload;
+import org.hibernate.beanvalidation.tck.tests.methodvalidation.model.StockItemRecord;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.METHOD;
+
+/**
+ * @author Gunnar Morling
```

Comment:
I'm guessing this is left from the ValidStockItem class, but it should probably either be removed or switched to your name

Reply:
Fixed